tests/unit/deprecation-collector-test.js

```diff
@@ -55,6 +55,7 @@ window.deprecationWorkflow.config = {
 };`);
 });
 
+// Fails in Ember 3.5+
 test('calling flushDeprecations without ID returns full message plus an additional deprecation for missing ID', (assert) => {
```

Comment:
Lets guard these tests in Ember >= 3.

Specifically:

```js
import hasEmberVersion from '@ember/test-helpers/has-ember-version';

if (!hasEmberVersion(3,0)) {
  // tests here 
}
```

Reply:
done, tests guarded.